.github/renovate.json

```
@@ -2,5 +2,16 @@
   "extends": [
     "config:base"
   ],
-  "commitBody": "Signed-off-by: Renovate Bot <bot@renovateapp.com>"
+  "commitBody": "Signed-off-by: Renovate Bot <bot@renovateapp.com>",
+  "regexManagers":[
+    {
+      "fileMatch":[
+        "^\\.github\\/workflows\\/.*\\.yml"
+      ],
+      "matchStrings":[
+        "terraform_version: (?<currentValue>.*?)  # renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\n"
```

```suggestion
        "terraform_version: (?<currentValue>.*?)\\s*# renovate: datasource=(?<datasource>.*?) depName=(?<depName>.*?)\n"
```

Just a very minor tweak to not require exactly two whitespaces, but any number of them. Did no testing, but should work.

---

Looks fine to me, otherwise!